<launch>

    <!-- launching the node for building and publishing the octomap -->
    <node name="octomap_pa_node" pkg="octomap_pa"
      type="octree_pa_node" output="screen">
        <!-- load parameter -->
        <rosparam command="load"
          file="$(find octomap_pa)/config/parameter.yaml" />
    </node>
</launch>
